modules/sb-start-tsb.adoc

```diff
@@ -15,32 +15,49 @@ After you have installed the {tsb-name} Operator, you can start the
 
 .Procedure
 
-. Navigate in the web console to *Operators* -> *Installed Operators* and select the `openshift-template-service-broker` project.
+. Using the *Administrator* perspective, navigate in the web console to
+*Operators* -> *Installed Operators* and select the
+`openshift-template-service-broker` project.
 . Select the *Template Service Broker Operator*.
-. Under *Provided APIs*, click *Create New* for *Template Service Broker*.
+. Under *Provided APIs*, click *Create Instance* for *Template Service Broker*.
 . Review the default YAML and click *Create*.
-. Verify that the {tsb-name} has started.
+. Verify that the {tsb-name} starts correctly by checking that the template
+applications are available.
 +
-After the {tsb-name} has started, you can view the available
-template applications by navigating to *Catalog* -> *Developer Catalog* and
-selecting the *Service Class* checkbox. Note that it may take a few minutes for
-the {tsb-name} to start and the template applications to be
-available.
+--
+* To check from the web console, switch to the *Developer* perspective, then click
+*Add+* -> *From Catalog* to navigate to the *Developer Catalog* page. The
+template applications show up as *Template* type objects.
```

Comment:
FWIW, these aren't showing up for me right now in this spot. I _can_ view them from the Administrator perspective via Service Catalogs -> Broker Management -> Service Classes, however.

Reply:
Checked with Eng and changed this to the Admin view version.